bitbots_dynup/src/dynup_engine.cpp

```diff
@@ -224,48 +214,13 @@ void DynupEngine::start(bool front, geometry_msgs::Pose l_foot_pose, geometry_ms
    *    - move arms in finish position
    */
```

Comment:
In my opinion this comment explains quite well how `calculateSplines()` or maybe this engine in general operates. Please move it somewhere where it is more useful.